doc/src/build/wallet.md

```diff
@@ -503,16 +506,15 @@ OPTIONS:
 To view the object, use the following command:
 
 ```bash
-$ wallet object --id EEA4167BE074537F4A2879C7781D8EF4FFD651CC
+$ wallet object --id 0x124bbde643189b573c98d05c092f4927225421d7
 ```
 
 This should give you output similar to the following:
 
 ```shell
-Owner: AddressOwner(k#66af3898e7558b79e115ab61184a958497d1905a)
-Version: 0
-ID: EEA4167BE074537F4A2879C7781D8EF4FFD651CC
-Readonly: false
+ID: 0x124bbde643189b573c98d05c092f4927225421d7
+Version: 1
+Owner: Account Address ( 0x62cd5bc220b28a34265bcb24995fb45a51d39832 )
```

Comment:
Is this spacing in parenthesis correct?
( 0x62cd5bc220b28a34265bcb24995fb45a51d39832 )